Extension/src/LanguageServer/configurations.ts

```diff
@@ -632,6 +632,7 @@ export class CppProperties {
             // Otherwise, the JSON.parse result will have the \<escape character> missing.
             readResults = readResults.replace(/\\/g, '\\\\');
             readResults = readResults.replace(/\\\\"/g, '\\"'); // Need to revert the change to \".
```

Comment:
Is it possible to combine with the first replacement of `\"` with the correct number of `\` instead of having it done on a separate call?

Reply:
Yeah -- I thought of that afterwards.

Reply:
Let me know if you see any issue with the new regex.